### DISCUSSION TITLE
rss.cheredeprince.net is down

### DISCUSSION THREAD

**Discussion by cowsay** [FreshRSS/FreshRSS#5861] [General]
Hi, months ago I found FreshRSS, I was lazy to find host for my own instance so I start using one of the free instances from https://freshrss.org/cloud-providers.html, **Chère de Prince** (https://rss.cheredeprince.net/).

It worked well, but now its down for week or so, without warning.

I'm sad mostly because I cannot even export my feed (that significantly changed since the start of use), and I don't have any backups (which is my bad)

I don't blame anyone, that's your right, but if you hear, can you at least give the ability to log in and export my opml?

**Comment by Alkarex** [comment]
I will try to e-mail the admin

**Reply by Alkarex** [reply]
And ping @laBecasse

**Reply by laBecasse** [reply]
Hello, 
I am sorry for the inconvenience. I have moved to my new appartment two weeks ago and since then I don't have internet at home, so Chère de Prince is down. I have good hope that I can restore it next week. If you are interested to join me when Chère de Prince is offline, I am on Mastodon (I publish news in French) : https://mastodon.social/@LaBecasse
